weaviate/collection/collection_base.py

```diff
@@ -122,16 +124,28 @@ def get(self) -> List[Tenant]:
 
 class _Config:
     """
-    Represents all the CRUD methods available on a collection's configuration specification within Weaviate. This class
-    should not be instantiated directly, but is available as a property of the `Collection` class under
+    Represents all the CRUD methods available on a collection's configuration specification within Weaviate.
+
+    This clasa should not be instantiated directly, but is available as a property of the `Collection` class under
     the `collection.schema` class attribute.
     """
 
+    __cached: Optional[Dict[str, Any]]
+
     def __init__(self, connection: Connection, name: str) -> None:
+        self.__cached = None
```

Comment:
why not have the type here?

Reply:
I specified the type in the class variable definition, will change for clarity